packages/highcharts-theme/src/examples/DonutChart.tsx

```diff
@@ -3,76 +3,30 @@ import { clsx } from "clsx";
 import Highcharts, { type Options } from "highcharts";
 import HighchartsReact from "highcharts-react-official";
 import { type FC, useRef } from "react";
+import { donutOptions } from "./dependencies";
 
 export interface DonutChartProps {
   patterns?: boolean;
+  options: Options;
 }
 
-const DonutChart: FC<DonutChartProps> = ({ patterns = false }) => {
+const DonutChart: FC<DonutChartProps> = ({
+  patterns = false,
+  options = donutOptions,
+}) => {
   const DonutRef = useRef<HighchartsReact.RefObject>(null);
```

Comment:
```suggestion
  const donutRef = useRef<HighchartsReact.RefObject>(null);
```
Nit, variables should be camel cased.